sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/QueryConstants.cs

```diff
@@ -19,6 +19,28 @@ internal static class QueryConstants
         public const string Top = "TOP";
         public const string Count = "COUNT";
 
-        // other query keywords added in this same fashion as they are implemented
+        public const string And = "AND";
+
+        public const string IsDefined = "IS_DEFINED";
+        public const string IsNull = "IS_NULL";
+        public const string StartsWith = "STARTSWITH";
+        public const string EndsWith = "ENDSWITH";
+        public const string IsOfModel = "IS_OF_MODEL";
+        public const string IsBool = "IS_BOOL";
+        public const string IsNumber = "IS_NUMBER";
+        public const string IsString = "IS_STRING";
+        public const string IsPrimative = "IS_PRIMATIVE";
+        public const string IsObject = "IS_OBJECT";
+
+        // Maps comparison operators represented alphabetically to respective symbolic representations.
+        public static Dictionary<string, string> ComparisonOperators = new Dictionary<string, string>()
+        {
+            { QueryComparisonOperator.Equal.ToString(), "=" },
```

Comment:
why not just change the Dictionary type to be `Dictionary<QuerycomparisonOperator, string>' so you don't have to do the ToString() on all of them.
The lookup will be easier too.